force-app/main/default/objects/Account/listViews/All_Manufactuers.listView-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please confirm list views on shared objects are visible to EDRD only as applicable @Sumancdey @deepakmulamalla Is there an agreement to have SAT see this item?